src/endpoints/authentication/cert_auth.cpp

```diff
@@ -7,9 +7,53 @@
 #include "ccf/service/tables/members.h"
 #include "ccf/service/tables/nodes.h"
 #include "ccf/service/tables/users.h"
+#include "ds/x509_time_fmt.h"
+#include "enclave/enclave_time.h"
 
 namespace ccf
 {
+  static inline bool is_cert_valid_now(
```

Comment:
NB: This is applied to `user_cert` and `member_cert` requests, but _NOT_ to `node_cert`. If a node ends up with an expired cert, I think we still need to be able to speak to it, for the few requests that are `node_cert`-authenticated (`/join`).